.evergreen/build_example_projects.sh

```diff
@@ -9,6 +9,9 @@ else
     DIR=shared
 fi
 
+. .evergreen/find_cmake.sh
+export CMAKE="$CMAKE"
```

Comment:
The Ubuntu 18.04 hosts do not appear to have `cmake` in the path. `compile.sh` and `install_c_driver.sh` both use this script to discover the path to cmake, so this follows suit.

Reply:
Acknowledged